src/coreclr/pal/tests/palsuite/file_io/WriteFile/test2/WriteFile.cpp

```diff
@@ -61,8 +61,8 @@ PALTEST(file_io_WriteFile_test2_paltest_writefile_test2, "file_io/WriteFile/test
     const char * testString = "The quick fox jumped over the lazy dog's back.";
     const int testStringLen = strlen(testString);
```

Comment:
Should this implicit size_t -> int cast be made explicit as well? It is interesting that the compiler is not complaining about this `strlen("constant string")`, but it is complaining about the other `strlen("constant string")`.

Reply:
Declaring it as `DWORD` also works without the explicit cast.

Reply:
I would not depend on it. In general, compilers complain about any implicit narrowing these days.